JSON-connect/src/main.js

```diff
@@ -25,12 +25,11 @@
  * @param {String} message The exception message
  */
 function sendUserError(message) {
+  console.log(message);
```

Comment:
Please remove any excess calls to console.log.

Reply:
done